src/Ip.php

```diff
@@ -324,10 +324,10 @@ public static function ip2long($ip)
      * @param long $dec IPv4 or IPv6 long
      * @return string If IP is valid returns IP string representation, otherwise ''.
      */
-    public static function long2ip($dec, $ipv6 = false)
```

Comment:
this is a breaking change and I suggest to leave it `false`

Reply:
So it is not recommended to use automatic IPv6 detection for this function, to not to break compatibility, as I understand?

Reply:
My bad, you already set that to false to avoid BC break. So, it can be merged. Thank you.